mbta_finder.py

```diff
@@ -9,49 +12,56 @@
 
 from urllib.request import urlopen
 import json
-
-
-# Useful URLs (you need to add the appropriate parameters for your requests)
-GMAPS_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json"
-MBTA_BASE_URL = "http://realtime.mbta.com/developer/api/v2/stopsbylocation"
-MBTA_DEMO_API_KEY = "wX9NwuHnZU2ToO7GmGR9uw"
-
-
-# A little bit of scaffolding if you want to use it
-
-def get_json(url):
-    """
-    Given a properly formatted URL for a JSON web API request, return
-    a Python JSON object containing the response to that request.
-    """
-    pass
-
-
-def get_lat_long(place_name):
-    """
-    Given a place name or address, return a (latitude, longitude) tuple
-    with the coordinates of the given place.
-
-    See https://developers.google.com/maps/documentation/geocoding/
-    for Google Maps Geocode API URL formatting requirements.
-    """
-    pass
-
+from pprint import pprint
+
+
+def get_url(place_name):
+	GMAPS_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json"
+	place_name= place_name.replace(" ", "+")
+	url = GMAPS_BASE_URL + "?address=" +place_name
+	f = urlopen(url)
+	response_text = f.read()
+	response_data= json.loads(str(response_text,"utf-8"))
+	return response_data
+
+def get_lat_long(response_data):
+	"""
+	Given a place name or address, return a (latitude, longitude) tuple
+	with the coordinates of the given place.
+
+	See https://developers.google.com/maps/documentation/geocoding/
+	for Google Maps Geocode API URL formatting requirements.
+	"""
+	lat = response_data["results"][0]["geometry"]["location"]["lat"]
+	lng = response_data["results"][0]["geometry"]["location"]["lng"]
+	return lat, lng
 
 def get_nearest_station(latitude, longitude):
-    """
-    Given latitude and longitude strings, return a (station_name, distance)
-    tuple for the nearest MBTA station to the given coordinates.
-
-    See http://realtime.mbta.com/Portal/Home/Documents for URL
-    formatting requirements for the 'stopsbylocation' API.
-    """
-    pass
+	"""
+	Given latitude and longitude strings, return a (station_name, distance)
+	tuple for the nearest MBTA station to the given coordinates.
+
+	See http://realtime.mbta.com/Portal/Home/Documents for URL
+	formatting requirements for the 'stopsbylocation' API.
+	"""
+	MBTA_BASE_URL = "http://realtime.mbta.com/developer/api/v2/stopsbylocation"
+	MBTA_DEMO_API_KEY = "wX9NwuHnZU2ToO7GmGR9uw"
+	url = MBTA_BASE_URL + "?api_key=" + MBTA_DEMO_API_KEY + "&lat=%s&lon=%s&format=json"%(latitude,longitude)
+	f = urlopen(url)
+	response_text = f.read()
+	response_data= json.loads(str(response_text,"utf-8"))
```

Comment:
I can't help but notice that every once in a while there just isn't a space before your equals signs